cmd/nvidia-ctk-installer/container/runtime/runtime.go

```diff
@@ -53,6 +53,11 @@ func Flags(opts *Options) []cli.Flag {
 			Destination: &opts.Config,
 			EnvVars:     []string{"RUNTIME_CONFIG", "CONTAINERD_CONFIG", "DOCKER_CONFIG"},
 		},
+		&cli.StringFlag{
+			Name:        "container-runtime-executable-path",
```

Comment:
Why not just `executable-path` or `runtime-executable-path`? This is consistent with how `socket` is named

Reply:
Yes, I think this may be better. Let me take another pass.